core/src/autogluon/core/hpo/ray_hpo.py

```diff
@@ -384,7 +384,7 @@ def get_resources_per_trial(
         self.num_parallel_jobs = resources_info.get('num_parallel_jobs', None)
         self.cpu_per_job = resources_info.get('cpu_per_job', None)
         self.gpu_per_job = resources_info.get('gpu_per_job', None)
-        self.resources_per_trial = resources_info.get('resources_per_trial', None)
+        self.resources_per_trial = resources_info.get('resources_per_job', None)
```

Comment:
May need to change the variable name here to `resources_per_job`.

Reply:
So in hpo, it is intended to be `resources_per_trial`. The resource calculator returns `resources_per_job` because it's  supposed to be general.